src/types/models.ts

```diff
@@ -73,6 +73,9 @@ export type BatteryStatus = "critical" | "low" | "good" | "full"
 
 export type CommonDeviceProperties = {
   name: string
+  model: {
```

Comment:
should this be optional?

Reply:
It always returns something for the display name, sometimes it's like `Unknown Model` etc if we don't know.